bricks-assemble/pom.xml

```diff
@@ -127,10 +127,9 @@
         </dependency>
     </dependencies>
 
-    <!-- FIXME Is it really a good idea to define this repository inside 
-        the pom? How about if we would like or need to overwrite it some day? Can 
-        we do it? Or is the repository area inside the pom the most binding place 
-        one can configure? -->
+    <!-- 
+        FIXME Is it really a good idea to define this repository inside the pom? How about if we would like or need to overwrite it some day? Can we do it? Or is the repository area inside the pom the most binding place one can configure? 
```

Comment:
is that in line with your max-line-length configuration used in the code?

Reply:
Nope. Forgot to format.

Reply:
:D